docs/pages/2020_BIGMAP/sections/qe.rst

```diff
@@ -323,7 +323,141 @@ Use this to produce a PDF of the band structure:
+Moreover, we once again *tag* this append step of our query with ``pw``.
+Let's have a look at how many nodes we have in the ``tutorial_pbesol`` group:
+
+.. code-block:: ipython
+
+    In [5]: qb.count()
+    Out[5]: 57
+
+Great, now let's figure out which structures are magnetic!
+Of course, the information we are interested in are the structures and their absolute magnetization, which we'll query for in the final two steps.
+First, we'll *append* the ``StructureData`` to the query:
+
+.. code-block:: ipython
+
+    In [6]: qb.append(StructureData, with_outgoing='pw', project='extras.formula');
+
+In this step, we've used the ``with_outgoing`` relationship to look for structures that have an *outgoing* link to the ``PwCalculations`` referenced with the ``pw`` tag.
+That means that from the ``PwCalculation``'s perspective, the ``StructureData`` is an input, exactly what we want.
+We also use the ``project`` keyword argument to *project* the formula of the structure, which for this tutorial has been stored in the ``extras`` of the ``StructureData`` nodes.
+By *projecting* the formula, it will be a part of the results of our query.
+
+The final ``append()`` call puts using *relationships*, *filters* and *projections* together.
+Here we are looking for the ``output_parameters`` ``Dict`` nodes, which are outputs of the ``PwCalculation`` nodes.
+However, we are only interested in structures for which the ``absolute_magnetization`` is larger than zero:
+
+.. code-block:: ipython
+
+    In [7]: qb.append(
+       ...:     Dict, with_incoming='pw', filters={'attributes.absolute_magnetization': {'>': 0.0}},
+       ...:     project='attributes.absolute_magnetization'
+       ...: );
+
+Let's go over the arguments again:
+
+    * The first *positional* argument tells the ``QueryBuilder`` we want to append ``Dict`` nodes to our query.
+    * ``with_incoming`` indicates there is an incoming link from a ``PwCalculation``, referenced by the ``'pw'`` tag.
+    * We're ``filter``-ing for magnetic structures, i.e. with ``absolute_magnetization`` above zero.
+    * Finally, we ``project`` the absolute magnetization so it is added to the list of our results for *each* query result.
+
+Our query is now complete!
+Let's have a look at the results:
+
+.. code-block:: ipython
+
+    In [8]: qb.all()
+    Out[8]:
+    [['LaMnO3', 3.5],
+     ['MnO3Sr', 3.15],
+     ['CoO3Sr', 2.42],
+     ['FeLaO3', 3.11],
+     ['CoLaO3', 1.13],
+     ['NiO3Sr', 0.77],
+     ['FeO3Sr', 3.38]]
+
+You can see that we've found 7 magnetic structures for the calculations in the ``tutorial_pbesol`` group, along with their formulas and magnetizations.
+We've set up a little script (:download:`demo_query.py <include/snippets/demo_query.py>`) that performs a similar query to obtain these results for all three groups, and then postprocess the data to make a simple plot.
```

Comment:
Should we also include here a preview of `demo_query.py`, like we do with the other demo scripts?

Reply:
👍  I'll add it in a dropdown!